### ISSUE TITLE
lyrics cmd doesnt give lyrics

### ISSUE THREAD

**Issue by cybermagma-dev** [nizewn/Dodong#39]
I have inserted the genius api but even then the bot doesnt give any lyrics even if i write the  song manually

**Comment by nizewn** [comment]
it's working fine for me. are you sure you're using the **client access token** from https://genius.com/api-clients?
![image](https://user-images.githubusercontent.com/48910077/188129996-443e77de-24b1-45f1-a96f-71102e039db2.png)

**Comment by cybermagma-dev** [comment]
> 

yeah i have inserted that only but then also for every song it says no lyrics like "-lyrics stay"

**Comment by Paartx** [comment]
It does'nt show the lyrics of a specific song, but only the lyrics of the current playing song.

**Comment by cybermagma-dev** [comment]
> It does'nt show the lyrics of a specific song, but only the lyrics of the current playing song.

I did try only lyrics Cmd but still its not showing

**Comment by cybermagma-dev** [comment]
> it's working fine for me. are you sure you're using the _**client access token**_ from https://genius.com/api-clients? ![image](https://user-images.githubusercontent.com/48910077/188129996-443e77de-24b1-45f1-a96f-71102e039db2.png)
> 
> if yes, can you give some examples of songs or queries where it doesn't work?

![image](https://user-images.githubusercontent.com/77920021/189717416-273b8866-ec94-4d40-bd1f-f73c4bc91a02.png)
